### DISCUSSION TITLE
Indiana jone and the infernal machine Single editor to view and import and export to facilitate the process would be welcome.

### DISCUSSION THREAD

**Discussion by Paulo422024** [smlu/ProjectMarduk#10] [General]
Hello. I am amazed by the work and effort put into the tools for this game which is hardly difficult to find.
My question is, wouldn't it be possible to create a single editor that can explore and visualize the files loads all 3d models, animations, textures, audio etc... in a single editor and export everything to an obj or fbx file to import into 3d engines, in my case in unity ?!.
Because that helps a lot. My goal is to recreate the game in unity with physics etc... As well as making the 3d models even better. Thanks.

**Comment by smlu** [comment]
Hey, you should definitively check out this PR: https://github.com/r1sc/OpenSith/pull/6
Also fyi, there is already work in progress by mod community to recreate the engine in [godot](https://godotengine.org) and port what was made in Unity to it. And some member of the community is already working on HD upgrade of models. (join discord server)

Now to answer your question, unfortunately atm I don't have time to work any fuhrer on these tools (working already on level editor and some other projects). Tho with some effort there is already option now to do what you want to achieve with existing tools. 
i.e.:
 - cndtool can convert level geometry to OBJ format in batch
 - blender can convert 3dos to obj/fbx. Tho you'll have to find or create script to do in batch.
